BusyList/Handlers/AddHandler.cs

```diff
@@ -0,0 +1,26 @@
+﻿using BusyList.Commands;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BusyList.Handlers
+{
+    public class AddHandler : IHandler<AddCommand>
+    {
+        private ITaskRepository _taskRepository;
```

Comment:
we can make this "readonly" since we are never gonna create a new task repository inside this class